cli/command/test_helpers.go

```diff
@@ -17,6 +17,18 @@ type TestCommandBase struct {
 	Resolver *mock_resolver.MockResolver
 }
 
+func NewTestCommand(t *testing.T) (*TestCommandBase, *gomock.Controller) {
```

Comment:
doesn't this exist somewhere already in the `cli-refactor` branch?

Reply:
Oop, yeah, actually `newTestCommand()` exists right below this. I had created this public function when I thought that the `client` package would need to use things from the `command` package, and had forgotten to take this out. Nice catch. ✅